### ISSUE TITLE
How to avoid the influence of color in RGB?

### ISSUE THREAD

**Issue by OOF-dura** [thmoa/tex2shape#10]
The paper takes advantage of  'rich illumination and shading information contained in RGB values'. But will the predicted result be related to color? (For example, the network can rebuild a person in white clothes but can't handle people who wear black clothes well) If so, how does the network solve it?

Thanks!

**Comment by thmoa** [comment]
like all deep learning based works: variance in the training data

**Comment by OOF-dura** [comment]
> like all deep learning based works: variance in the training data

Thanks! Does that mean during training, you just used RGB images without any preprocessing?

Thx again!

**Comment by thmoa** [comment]
Please refer to the paper...

**Comment by OOF-dura** [comment]
> Please refer to the paper...

OK. But I find when you map the color in image to the UV map based on the result of densepose, the color of the skin is a little blue. 
![image](https://user-images.githubusercontent.com/22954901/83943411-6c783280-a82e-11ea-8348-c79dc144bc66.png)
![image](https://user-images.githubusercontent.com/22954901/83943495-35565100-a82f-11ea-98c4-9d11de152975.png)

which is different from the picture in the paper:
![image](https://user-images.githubusercontent.com/22954901/83943479-10fa7480-a82f-11ea-8670-9103685f570a.png)

Thanks!

**Comment by thmoa** [comment]
RGB vs BGR.

**Comment by OOF-dura** [comment]
Thanks!

**Comment by lemon1220** [comment]
>   @OOF-dura @Thanks!
hello, can you kindly teach me how to gernerate the UVmap just like the following ?
![image](https://user-images.githubusercontent.com/52917659/110567134-ff624680-818b-11eb-943d-2899d7254976.png)
